docs: update is trusted (#30923)
mdn/content: files/en-us/web/api/event/istrusted/index.md

The **`isTrusted`** read-only property of the
{{domxref("Event")}} interface is a boolean value that is `true`

> when the event was generated by a user action or user agent,
> and `false` when the event was dispatched via

Reviewer: ```suggestion
when the event was generated by user agent,
and `false` when the event was dispatched via
```

following up the spec, maybe there only mention *user agent* is enough?

Author: Sure, I will follow tha.t

> when the event was generated by the user agent (including using programmatic methods, such as {{domxref("HTMLElement.focus()")}}),

Reviewer 2: ```suggestion
when the event was generated by the user agent (including via user actions and programmatic methods such as {{domxref("HTMLElement.focus()")}}),
```

{{domxref("EventTarget.dispatchEvent()")}}.

> The only exception is click, which intialise the **`isTrusted`**
> property to false in user agents.

Reviewer: ```suggestion
The only exception is `click` event, which intialise the **`isTrusted`**
property to `false` in user agents.
```

> The only exception is `click` event, which intialise the **`isTrusted`**

Reviewer 2: ```suggestion
The only exception is the `click` event, which initializes the `isTrusted`
```
